components/emoji.js

```diff
@@ -0,0 +1,30 @@
+import { memo, useState, useEffect } from 'react'
+import { LazyLoadImage } from 'react-lazy-load-image-component'
+
+const Emoji = ({ name }) => {
+    let [image, setImage] = useState()
+    useEffect(() => {
+        try {
+            fetch("https://emoji-ranker.now.sh/custompng")
+                .then(r => r.json())
+                .then(emojis => {
+                    if (emojis[name.replaceAll(":", "")]) {
+                        setImage(emojis[name.replaceAll(":", "")])
+                        return
+                    }
+                    setImage("https://emoji.slack-edge.com/T0266FRGM/parrot/c9f4fddc5e03d762.gif")
+                })
+        } catch (e) { alert("POOPIE") }
```

Comment:
lol wut, sounds super cool!

Reply:
Oof lol that was for error checking, I'll probably delete that